portal/portal-api/api/src/java/org/sakaiproject/portal/api/PortalService.java

```diff
@@ -332,4 +343,46 @@
 	 */
 	public List<Map> getQuickLinks(String siteSkin);
 
-}
\ No newline at end of file
+	/**
```

Comment:
I'd be temped to move this to it's own interface rather than increasing the surface area of the PortalService, maybe have `org.sakaiproject.portal.api.BullhornService`?